example/cfis_simu/mask_default/MEGAPRIME_star_i_13.8.reg

```diff
@@ -0,0 +1,24 @@
+-11.5 68
```

Comment:
I think some of those files used for masking (e.g. Messier & NGC objects) are not used.